### DISCUSSION TITLE
Fix issue regarding going to the top of profile after pressing back

### DISCUSSION THREAD

**Discussion by ShhubhamA** [deso-protocol/dips#56] [Medium CIPs]
So when you scroll down on someone's profile to see the previous content they posted and then you like a particular post and click to see the comments and else ! And when you presses back, you go to the top of the profile and then you have to again scroll down again!

This gets annoying at times when you want to binge on someone profile

**Comment by ItsAditya-xyz** [comment]
agreed! i would like to continue from the same point while going back.

**Comment by lazynina** [comment]
This would be great! I support someone resolving this issue. @AdityaChaudhary0005 would you like to try to fix it?

**Reply by ItsAditya-xyz** [reply]
I wish I was good at Angular 😅
Though I can try..

**Reply by tijno** [reply]
@AdityaChaudhary0005 The back button works ok in the feed, but not in the notifications.

i looked at this a while back to see if i could get back button to work for notifications - the issue there was that it reloads notification when you go back.

`scrollPositionRestoration` in the Router extra options may be the solution.

Hopefully this helps.
